### DISCUSSION TITLE
Regarding Alignment of the content

### DISCUSSION THREAD

**Discussion by LShivaRudra** [MrGreensWorkshop/MrGreen-JekyllTheme#210] [How to] [answered]
Hi, I would like to know about how I can make the content center-aligned rather than left-aligned as shown in the pics attached(since a lot of empty space is there at the right).

TIA

![Screenshot from 2023-11-12 13-43-05](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/assets/81946419/a88e81b9-c579-46e8-a3a5-f7104adb0447)
![Screenshot from 2023-11-12 13-43-30](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/assets/81946419/2f081772-320e-448f-8f47-7f1ccff8d199)

**Comment by MrGreensWorkshop** [comment, marked as answer]
Not just for this particular problem, but for all things you want to change, use a proper editor like VS Code so you can search for texts.
Next, use your web browser development tools. This is not just for you. (For the people struggling to change things.)
- Download the VS Code,
- Learn how to use web development tools. For Chrome and Firefox, press F12.

For the things you want to change.
- Check out the yml files in `_data/conf/` because settings are stored there. If you can't find the setting or option, use the search function in VS Code and look for it.
- If you can't find it, open your web browser's web development tool (F12) and look for items you want to change.
![before](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/assets/76992231/93419f84-71e9-4328-b221-8404f4375e3e)

So we found the container (div), which holds components.

- You can experiment with the numbers; this is not centering; you need to calculate the left margin, but you also need to account for the side panel (visible or not).
![after](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/assets/76992231/52e265b0-4188-41c1-b0af-f2c78f1cd95a)

- Finally, after you find the correct values for the things you want to change, search for the things and change the values. (You also need to do same things with the footer.)

![vscode](https://github.com/MrGreensWorkshop/MrGreen-JekyllTheme/assets/76992231/cd329234-6a38-4e2c-bb6d-4ddb86bf4cbc)

- Or you can put the `main-container` inside another container (div) try to center it.

This is how I develop the template. I hope you can accomplish what you want to do.

**Reply by LShivaRudra** [reply]
Thanks!
